signac/synced_collections/backends/collection_zarr.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""Implements a Zarr SyncedCollection backend."""
+from copy import deepcopy
+
+from .. import SyncedAttrDict, SyncedCollection, SyncedList
+
+try:
+    import numcodecs
+
+    ZARR = True
+except ImportError:
+    ZARR = False
+
+
+class ZarrCollection(SyncedCollection):
```

Comment:
Skipping review of this file for now. Several comments from other backends need to be applied here too.